starknet_py/contract_abi_resolver.py

```diff
@@ -0,0 +1,131 @@
+# Needed because of string typed Contract
```

Comment:
This should be placed in some separate module along with proxy_checks instead of top level directory.